### DISCUSSION TITLE
How to render two models at the same time

### DISCUSSION THREAD

**Discussion by dugufly** [google/filament#5526] [Q&A]
I want to render two models at the same time, one of them needs to be hidden, does not refresh the color buffer, only affects the depth buffer; the other model is displayed normally.

I have two questions:
1. How to render two models at the same time?
2. How to close the color buffer, the corresponding function of opengl is: glColorMask(GL_FALSE, GL_FALSE, GL_FALSE, GL_FALSE);

**Comment by bejado** [comment]
1. Create a `Renderable` component for each model and add them both to a `Scene`.
2. Create two separate materials. On one, set the [`colorWrite`](https://google.github.io/filament/Materials.html#materialdefinitions/materialblock/rasterization:colorwrite) material property to false.
